frontend/src/store.tsx

```diff
@@ -7,7 +7,7 @@ import logger from 'redux-logger'
 const plugins = [immerPlugin(), selectPlugin()]
 let middlewares: any[] = []
 
-if (process.env.NODE_ENV !== 'test') middlewares.push(logger)
+// if (process.env.NODE_ENV !== 'test') middlewares.push(logger)
```

Comment:
Intentional?

Reply:
No didn't mean to commit - backed it out already. Thanks!